From 6a27c0f9b235fdb17b09d934cd9b42a91972efe7 Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Sat, 25 Jul 2026 15:52:59 -0400
Subject: [PATCH 230/245] version bump EXTENDNO to Reaper_v1.7.9

v1.7.9 = d48c013de2 (2.4GHz hex fix for MLO-excluded band) + f237fd351c (VPN Fusion device-glyph CSS scope fix) + 986eb9f041 (speed test silent auto-retry).
---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index a1587d13ce..9b18ff18d8 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v1.7.8
+EXTENDNO=Reaper_v1.7.9
 
 RCNO=0
-- 
2.25.1

